From 8533f03b6e6830704d3ff4195e8224a68a39763b Mon Sep 17 00:00:00 2001
From: Igor Grinberg <grinberg@compulab.co.il>
Date: Mon, 4 Mar 2013 12:30:35 +0200
Subject: ARM: i.MX6: cm-fx6: move GPU and ASRC pdata

Move GPU and ASRC pdata closer to its use.
No functional changes.

Signed-off-by: Igor Grinberg <grinberg@compulab.co.il>
---
 arch/arm/mach-mx6/board-cm-fx6.c |   16 +++++++++-------
 1 file changed, 9 insertions(+), 7 deletions(-)

diff --git a/arch/arm/mach-mx6/board-cm-fx6.c b/arch/arm/mach-mx6/board-cm-fx6.c
index 8539476..7531025 100644
--- a/arch/arm/mach-mx6/board-cm-fx6.c
+++ b/arch/arm/mach-mx6/board-cm-fx6.c
@@ -641,9 +641,6 @@ static void __init cm_fx6_init_usb(void)
 	mx6_usb_dr_init();
 }
 
-static struct viv_gpu_platform_data imx6_gpu_pdata __initdata = {
-	.reserved_mem_size = SZ_64M,
-};
 
 /* HW Initialization, if return 0, initialization is successful. */
 static int mx6_arm2_sata_init(struct device *dev, void __iomem *addr)
@@ -729,10 +726,6 @@ static struct ahci_platform_data mx6_arm2_sata_data = {
 	.exit	= mx6_arm2_sata_exit,
 };
 
-static struct imx_asrc_platform_data imx_asrc_data = {
-	.channel_bits	= 4,
-	.clk_map_ver	= 2,
-};
 
 static void cm_fx6_hdmi_init(int ipu_id, int disp_id)
 {
@@ -1091,6 +1084,15 @@ static void __init cm_fx6_init_led(void)
 static inline void cm_fx6_init_led(void) {}
 #endif
 
+static struct imx_asrc_platform_data imx_asrc_data = {
+	.channel_bits	= 4,
+	.clk_map_ver	= 2,
+};
+
+static struct viv_gpu_platform_data imx6_gpu_pdata __initdata = {
+	.reserved_mem_size = SZ_64M,
+};
+
 /*
  * Board specific initialization.
  */
-- 
1.7.9.5

